presto-accumulo/src/test/java/com/facebook/presto/accumulo/TestAccumuloTpchQueries.java

```diff
@@ -0,0 +1,222 @@
+    public void cleanup()
+    {
+        dropTpchTables(runner, runner.getDefaultSession());
+    }
+
+    @Test
+    public void testQuery1()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/1.sql");
+    }
+
+    @Test
+    public void testQuery2()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/2.sql");
+    }
+
+    @Test
+    public void testQuery3()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/3.sql");
+    }
+
+    @Test(expectedExceptions = RuntimeException.class)
+    public void testQuery4()
+            throws Exception
+    {
+        // not yet implemented: com.facebook.presto.sql.tree.ExistsPredicate
+        runQuery("src/test/resources/tpch/4.sql");
+    }
+
+    @Test
+    public void testQuery5()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/5.sql");
+    }
+
+    @Test
+    public void testQuery6()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/6.sql");
+    }
+
+    @Test
+    public void testQuery7()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/7.sql");
+    }
+
+    @Test
+    public void testQuery8()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/8.sql");
+    }
+
+    @Test
+    public void testQuery9()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/9.sql");
+    }
+
+    @Test
+    public void testQuery10()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/10.sql");
+    }
+
+    @Test
+    public void testQuery11()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/11.sql");
+    }
+
+    @Test
+    public void testQuery12()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/12.sql");
+    }
+
+    @Test(expectedExceptions = RuntimeException.class)
+    public void testQuery13()
+            throws Exception
+    {
+        // Non-equi joins only supported for inner join: (NOT ("o"."comment" LIKE '%special%requests%'))
+        runQuery("src/test/resources/tpch/13.sql");
+    }
+
+    @Test
+    public void testQuery14()
+            throws Exception
+    {
+        runQuery("src/test/resources/tpch/14.sql");
+    }
+
+    @Test
+    public void testQuery15()
+            throws Exception
+    {
+        runner.execute("create view revenue as " +
+                "select " +
+                "l.suppkey AS number, " +
+                "sum(l.extendedprice * (1 - l.discount)) AS revenue " +
+                "from " +
+                "lineitem l " +
+                "where " +
+                "l.shipdate >= date '1996-01-01' " +
+                "and l.shipdate < date '1996-01-01' + interval '3' month " +
+                "group by " +
+                "l.suppkey");
+
+        runQuery("src/test/resources/tpch/15.sql");
+
+        runner.execute("DROP VIEW revenue");
```

Comment:
Put this in `finally` block. Make sure to log error on exception while in tearDown block.